packages/env/package.json

```diff
@@ -35,6 +35,7 @@
 	"exports": {
 		".": "./lib/env.js",
 		"./lib/cli.js": "./lib/cli.js",
+		"./lib/config": "./lib/config/index.js",
```

Comment:
What is this for? Just noting the inconsistency with the file extension included for `cli.js`. Ideally we could call this like `/cli` and `/config` if needed, but I guess that ship has sailed.